src/main/kotlin/net/lumalyte/lg/infrastructure/services/apollo/GuildTeamService.kt

```
@@ -299,48 +292,47 @@ class GuildTeamService(
+            val members = memberService.getGuildMembers(guildId)
+            cachedGuildMembers[guildId] = members.map { it.playerId }
+            cachedTrackingEnabled[guildId] = guildService.getGuild(guildId)?.trackingEnabled ?: true
+
             refreshGuildTeam(guildId)
 
-            // If player is on LC, show them their team
             val player = Bukkit.getPlayer(playerId) ?: return
             if (!lunarClientService.isLunarClient(player)) return
-
             createGuildTeam(guildId)
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)